package.json

```diff
@@ -2,14 +2,13 @@
   "name": "pokemon-showdown",
   "description": "The server for the Pokémon Showdown battle simulator",
   "version": "0.11.7",
-  "main": ".sim-dist/index.js",
+  "main": "dist/sim/index.js",
```

Comment:
this is off-topic, but why is `sim/index` the entry point for what's described as the *server*?

Reply:
It's for the node package, which is meant to export simulation tools.

Reply:
then the description is wrong